components/ExampleNav/ExampleNavContainer.coffee

```diff
@@ -4,6 +4,10 @@
 ExampleNav                     = require './ExampleNav'
 
 navs =
+  Dropdowns: [
+    'NavbarDropdownExamples'
```

Comment:
Also, can you please merge changes from dev into your branch? it would eliminate most of the conflicts at the time of merging PR. In fact, as de facto, you can always merge latest changes from dev into your feature branches to be updated with the code.